.github/workflows/image.yml

```
@@ -0,0 +1,54 @@
+name: image
+on:
+  pull_request:
+  push:
+    branches: [master]
```

```suggestion
    branches:
      - master
      - release/**
```

This should match the branches that are built for deployment